ansible/roles/ocp-workload-aiadv-templates/ilt_provision.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+END_PROJECT_NUM=1
+START_PROJECT_NUM=1
+WORKLOAD="ocp-workload-fuse-ignite"
```

Comment:
same here

Reply:
@fridim Thanks, addressed using this commit 20f845963ea22fe8157cb7bee7503a8f8a564a6c